requirements/edx/base.txt

```diff
@@ -46,7 +46,7 @@ edx-drf-extensions==1.2.2
 edx-lint==0.4.3
 edx-django-oauth2-provider==1.1.4
 edx-django-sites-extensions==2.1.1
-edx-enterprise==0.9.0
+edx-enterprise==0.14.0
```

Comment:
That's a big bump!  I didn't realize that we were several versions behind for this in edx-platform.